pytest.ini

```diff
@@ -0,0 +1,5 @@
+# pytest.ini
+[pytest]
+minversion = 5.0
+addopts = -ra -q
+testpaths =
```

Comment:
shouldn't there be some testpath defined here or is this line intentionally incomplete?